testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/OperationWarningTestsCase.java

```diff
@@ -0,0 +1,131 @@
+            PathElement.pathElement(SUBSYSTEM, "io"), PathElement.pathElement(WORKER, NAME_WORKER));
+    protected static final PathAddress ADDRESS_REMOTING = PathAddress.pathAddress(
+            PathElement.pathElement(PROFILE, NAME_PROFILE), PathElement.pathElement(SUBSYSTEM, "remoting"),
+            PathElement.pathElement("configuration", "endpoint"));
+    protected static final String BAD_LEVEL = "X_X";
+
+    @Before
+    public void before() throws Exception {
+        testSupport = DomainTestSuite.createSupport(DatasourceTestCase.class.getSimpleName());
+        domainMasterLifecycleUtil = testSupport.getDomainMasterLifecycleUtil();
+        domainSlaveLifecycleUtil = testSupport.getDomainSlaveLifecycleUtil();
+        addWorker();
+    }
+
+    @After
+    public void after() {
+        try {
+            setRemotingWorkerTo("default", "OFF");
+        } catch (Exception e) {
+        }
+
+        try {
+            removeWorker();
+        } catch (Exception e) {
+        }
+        testSupport = null;
+        domainMasterLifecycleUtil = null;
+        domainSlaveLifecycleUtil = null;
+        DomainTestSuite.stopSupport();
+    }
+
+    @Test
+    public void testMe() throws Exception {
+         ModelNode result = setRemotingWorkerTo(NAME_WORKER,BAD_LEVEL);
+         Assert.assertTrue(result.toString(), Operations.isSuccessfulOutcome(result));
+         result = result.get(RESULT);
+         Assert.assertTrue(result.toString(),result.hasDefined(RESPONSE_HEADERS));
+         ModelNode responseHeaders = result.get(RESPONSE_HEADERS);
+         Assert.assertTrue(responseHeaders.hasDefined(WARNINGS));
+         List<ModelNode> warnings = responseHeaders.get(WARNINGS).asList();
+         Assert.assertTrue(warnings.size() == 2);
+         ModelNode warningLoggerLevel = warnings.get(0);
+         String message = warningLoggerLevel.get(WARNING).asString();
+         Assert.assertEquals(ControllerLogger.ROOT_LOGGER.couldntConvertWarningLevel(BAD_LEVEL), message);
+         Level level = Level.parse(warningLoggerLevel.get(LEVEL).asString());
+         Assert.assertEquals(Level.ALL,level);
+         ModelNode warningWorker = warnings.get(1);
+         message = warningWorker.get(WARNING).asString();
+         Assert.assertEquals(RemotingLogger.ROOT_LOGGER.warningOnWorkerChange(NAME_WORKER), message);
+         level = Level.parse(warningWorker.get(LEVEL).asString());
+         Assert.assertEquals(Level.WARNING,level);
+         //default level is "WARNING, set to severe and check if there are warnings
+         result = setRemotingWorkerTo("default","SEVERE").get(RESULT);
+         responseHeaders = result.get(RESPONSE_HEADERS);
+         Assert.assertFalse(responseHeaders.hasDefined(WARNINGS));
+         result = setRemotingWorkerTo("default","OFF").get(RESULT);
+         responseHeaders = result.get(RESPONSE_HEADERS);
+         Assert.assertFalse(responseHeaders.hasDefined(WARNINGS));
```

Comment:
I believe the "server-groups" structure in the response will also have headers from the relevant servers. This should assert that too, at least for one server.

AIUI the response doesn't include anything specific to any of the slave HCs. If so, fine. If not, then whatever comes out should be asserted in the test to make sure it doesn't change unintentionally.